samples/Samples.h

```diff
@@ -45,6 +45,7 @@ extern "C" {
 #define IOT_CORE_PRIVATE_KEY         ((PCHAR) "AWS_IOT_CORE_PRIVATE_KEY")
 #define IOT_CORE_ROLE_ALIAS          ((PCHAR) "AWS_IOT_CORE_ROLE_ALIAS")
 #define IOT_CORE_THING_NAME          ((PCHAR) "AWS_IOT_CORE_THING_NAME")
+//#define IOT_CORE_ENABLE_CREDENTIALS  1
```

Comment:
Can we add a note here that you need to uncomment this line to enable iot credentials?